boot/freeldr/freeldr/CMakeLists.txt

```diff
@@ -199,8 +197,8 @@ set(PCH_SOURCE
 add_pch(freeldr_common include/freeldr.h PCH_SOURCE)
 add_dependencies(freeldr_common bugcodes asm xdk)
 
+## GCC builds need this extra thing for some reason...
```

Comment:
Yes, if you remove that block GCC builds will complain. If you know a better way of doing this, please let me know.

Reply:
Someone invented that mini hal thing for whatever reasons. But IIRC it makes the MSVC build too large, so it's not used in MSVC builds, which also removes some extra features somewhere.